app/assets/src/components/views/user_preferences_view.scss

```diff
@@ -0,0 +1,27 @@
+@import "~styles/themes/elements";
```

Comment:
Is Jenn going to review this screen in general? or was it already in the mocks / discussed

Reply:
This is going to be admin-only for now, so no need to review.